src/app.ts

```diff
@@ -29,4 +41,13 @@ app.listen(3000, () => {
     console.log('Server started on port 3000');
 });
 
-app.get('/', getAllDatabases);
\ No newline at end of file
+app.get('/', getAllDatabases);
+
+app.get('/sales-employees', getAllSalesEmployees);
+app.get('/sales-employees/create', getCreateSingleSalesEmployee);
+app.post('/sales-employees/create', postCreateSingleSalesEmployee);
+app.get('/sales-employees/:id', getSingleSalesEmployee);
+app.get('/sales-employees/delete/:id', getSingleSalesEmployeeToDelete);
+app.post('/sales-employees/delete/:id', deleteSingleSalesEmployee);
+app.get('/sales-employees/edit/:id', getSingleSalesEmployeeToEdit);
```

Comment:
why is this needed?

Reply:
https://stackoverflow.com/questions/8054165/using-put-method-in-html-form